Dockerfile.gcc

```diff
@@ -0,0 +1,18 @@
+FROM gcc
+
+RUN gcc --version
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
+
+WORKDIR /app
+
+COPY . /app
+
+RUN rm -rf build
+RUN rm -rf node_modules
+RUN rm -f package-lock.json
+
+RUN npm install --unsafe-perm
+
+CMD ["node", "examples/index", "host.docker.internal:2181"]
```

Comment:
Maybe we should add some docker-compose file that runs all integration test suites?

Reply:
Great idea! Maybe in a separate PR? I'll add an issue for it.